docs/apis/table.md

```diff
@@ -606,6 +606,28 @@ Table result = in.orderBy("a.asc");
       </td>
     </tr>
 
+    <tr>
+      <td><strong>Offset</strong></td>
+      <td>
+        <p>Similar to a SQL OFFSET clause. Returns rows from offset position. It is technically part of the ORDER BY clause.</p>
+{% highlight java %}
+Table in = tableEnv.fromDataSet(ds, "a, b, c");
+Table result = in.orderBy("a.asc").offset(3);
+{% endhighlight %}
+      </td>
+    </tr>
+    
+    <tr>
+      <td><strong>Fetch</strong></td>
+      <td>
+        <p>Similar to a SQL FETCH clause. Returns a set number of rows. FETCH can’t be used by itself, it is used in conjunction with OFFSET.</p>
```

Comment:
I prefer `orderBy().limit()` as it's cleaner.  And then we can combine `Offset` and `Fetch` with `Limit`(maybe) instead. The logical in Offset and Fetch is a little duplicate.